src/paymentservice/logger.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const pino = require('pino');
+
+module.exports = pino({
+  name: 'paymentservice-server',
+  messageKey: 'message',
+  formatters: {
+    level (logLevelString, logLevelNum) {
+      return { severity: logLevelString }
+    }
+  }
+});
```

Comment:
**Thought:** We should use this in [`src/paymentservice/charge.js`](https://github.com/GoogleCloudPlatform/microservices-demo/blob/8bf9151b3fe6014cb3683bf39bcd02d6c418df63/src/paymentservice/charge.js#L19C7-L19C13) as well.